src/Dev/DevBuildController.php

```diff
@@ -39,4 +51,29 @@ public function build(HTTPRequest $request): HTTPResponse
             return $response;
         }
     }
+
+    public function canInit(): bool
+    {
+        return (
+            Director::isDev()
+            // We need to ensure that DevelopmentAdminTest can simulate permission failures when running
+            // "dev/tasks" from CLI.
+            || (Director::is_cli() && DevelopmentAdmin::config()->get('allow_all_cli'))
+            || Permission::check("ADMIN")
+            || Permission::check("ALL_DEV_ADMIN")
+            || Permission::check("CAN_DEV_BUILD")
+        );
+    }
+    
+    public function providePermissions(): array
+    {
+        return [
+            'CAN_DEV_BUILD' => [
+                'name' => _t(__CLASS__ . '.CAN_DEV_BUILD_DESCRIPTION', 'Can execute a Dev/Build'),
+                'help' => _t(__CLASS__ . '.CAN_DEV_BUILD_HELP', 'Can execute a Dev/Build (/dev/build).'),
```

Comment:
(just thinking out loud, feel free to disregard)

Somehow the 'Can execute a Dev/Build' is pulling my ears, but I can't think of a better way.. deb/build command? Or perhaps 'Can execute the build command'? We all know it's on the dev controller, and therefore it became known as such.. if we look at e.g. the tasks command, listing all the tasks, we don't say dev/tasks when we talk about it..

```suggestion
                'name' => _t(__CLASS__ . '.CAN_DEV_BUILD_DESCRIPTION', 'Can execute dev/build'),
                'help' => _t(__CLASS__ . '.CAN_DEV_BUILD_HELP', 'Can run the build command (/dev/build).'),
```

Reply:
> if we look at e.g. the tasks command, listing all the tasks, we don't say dev/tasks when we talk about it..

I do say dev/tasks when I talk about it :p But I also agree that "a Dev/Build" feels a little clunky, especially since "(/dev/build)" is there too. I like "the build command (/dev/build)"

Reply:
Yeah nice, I agree. Must admit I'd forgotten to revisit those strings after my initial pass through 😅